training/utils.py

```diff
@@ -99,14 +102,15 @@ def validate_dataset(filepaths_and_text, dataset_directory, symbols):
     invalid_characters = set()
     wavs = os.listdir(dataset_directory)
     for filename, text in filepaths_and_text:
+        text=text.lower()
```

Comment:
Good thinking, I think it would be even better if we used the `clean_text` function here (as that is what is actually used when training runs)